src/screens/FinishLineMode.tsx

```diff
@@ -80,6 +81,21 @@ export default function FinishLineModeScreen({ navigation }: Props): React.React
 	const [loading, setLoading] = useState(true);
 	const isUnmounted = useRef(false);
 	const flatListRef = useRef<FlatList>(null);
+	const deviceType = useRef<"Phone"|"Tablet">("Phone");
+
+	// Get Device Type
+	useEffect(() => {
+		const getDeviceType = async (): Promise<void> => {
+			const type = await Device.getDeviceTypeAsync();
+			if (type === 1) {
```

Comment:
My only comment on this would be to eventually create an AppConstants file where the device types could be stored.  And then instead of "1", you could have it be 
```suggestion
			if (type === AppConstants.device_type_phone) {
```

Or something like that. Otherwise, this is working well and looks good!